code/parachain/frame/crowdloan-rewards/src/lib.rs

```diff
@@ -339,6 +359,32 @@ pub mod pallet {
 			Self::do_unlock(reward_accounts);
 			Ok(())
 		}
+
+		/// Deletes all accounts provided in the `deletions` vector. If the provided `Balance` does
+		/// not match the current balance of the account, the deletion will fail.
+		#[pallet::weight(<T as Config>::WeightInfo::populate(deletions.len() as _))]
+		#[transactional]
+		pub fn delete(
```

Comment:
Not sure if we need two extrinsics here given that all we need is one transaction to correct the contributions as per original total

Reply:
Well we call them using batch, so it'll be 1 transaction.